runtime/src/main/java/org/corfudb/runtime/view/AbstractOrderedGUIDFactory.java

```diff
@@ -0,0 +1,18 @@
+package org.corfudb.runtime.view;
+
+/**
+ * Globally Unique Identity (GUID) generator that returns compact GUIDs having a notion of
+ * a comparable global ordering.
+ *
+ * Why not use Java UUIDs?
+ *   JAVA UUIDs lack cross node comparability or global ordering in the absence of time sync.
+ *   They are also 36 bytes long which can be an overkill for simple use cases.
+ *
+ * What about schemes like Twitter Snowflake?
+ *    Requires time synchronization between cluster nodes for ordering.
+ *
+ * Created by Sundar Sridharan on 5/22/19.
+ */
+public interface AbstractOrderedGUIDFactory {
```

Comment:
Don't use a sequence of more than two capital letters. Please rename to AbstractOrderedGuidFactory

Reply:
Done.